.github/workflows/coveralls.yml

```diff
@@ -14,7 +14,10 @@ jobs:
       run: |
         sudo apt-get -yq install lcov
     - name: config
-      run: CC=gcc ./config --banner=Configured --debug --coverage no-asm enable-fips enable-rc5 enable-md2 enable-ssl3 enable-nextprotoneg enable-ssl3-method enable-weak-ssl-ciphers enable-zlib enable-ec_nistp_64_gcc_128 no-shared enable-buildtest-c++ enable-external-tests -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION && perl configdata.pm --dump
+      run: CC=gcc ./config --banner=Configured --debug --coverage no-asm enable-fips enable-rc5 enable-md2 enable-ssl3 enable-nextprotoneg enable-ssl3-method enable-weak-ssl-ciphers enable-zlib enable-ec_nistp_64_gcc_128 no-shared enable-buildtest-c++ enable-external-tests -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+    - name: config dump
+      run: ./configdata.pm --dump
+
```

Comment:
Is there a reason for this sudden empty line? (not only here, you added it consistently)

Reply:
Readability.  It's a nice visual separation from the configuration and the "work".
I like paragraph breaks, they are easy on the eyes.

Reply:
To my eyes, this is similar to the line breaks we have between separate jobs, that made it easy to see what constitutes one job at a glance.  With these added line breaks, that's no longer true, you suddenly have to stop and look if the line break actually shows where a new job starts or if it's inside the job description.

On the other hand, I noticed that you added an empty line before the configuration steps in *some* jobs but not others...  so if you insist on this separation, please be consistent with the empty line before the configuration steps.

Reply:
Most of our CIs only have one job in them.

Reply:
And windows.yml does it differently.

Reply:
I'd agree here with @levitte that the extra empty lines you add with this change to the various CI configs should not be added. It does not seem to improve readability to me.